src/smart-components/role/add-role-permissions/add-role-permissions-summary-content.js

```diff
@@ -38,21 +45,34 @@ const AddRolePermissionSummaryContent = ({ selectedPermissions = [], role = {} }
             </Text>
           </GridItem>
           <GridItem span={9}>
-            <Text component={TextVariants.p}>
+            <TextContent component={TextVariants.p}>
               {selectedPermissions.map((permission, index) => (
                 <Text key={index}> {permission.uuid} </Text>
               ))}
-            </Text>
+            </TextContent>
           </GridItem>
+          {hasCostResources && (
+            <>
+              <GridItem span={3}>
+                <Text className="pf-c-title" component={TextVariants.h6}>
+                  Resource definitions:
+                </Text>
+              </GridItem>
+              <GridItem span={9}>
+                <TextContent component={TextVariants.p}>
+                  {resourceDefinitions
+                    .map((def) => def.resources)
+                    .map((resource, index) => (
```

Comment:
I don't think you need to have two maps.
```suggestion
                    .map(({ resources }, index) => (
```

Unless the `resources` are array, and if that's true then this won't help either. And you should import `flatten` from lodash.

Reply:
Yep, there are two arrays:
```
[{
  permission: 'str1'
  resources: ['res1', res2', ...]
}, {,...
}]
```

Reply:
What exactly is input and what is the expected output?